libs/providers/launchdarkly-client/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["../../../.eslintrc.json"],
-  "ignorePatterns": ["!**/*"],
+  "ignorePatterns": ["!**/*", "node_modules"],
```

Comment:
Did you get lint errors in the node_modules?

If so, maybe we should move this to the root config?

Reply:
this is the exact error I get: 
```
  Failed to load parser 'babel-eslint' declared in 'libs/providers/launchdarkly-client/node_modules/launchdarkly-js-sdk-common/.eslintrc.yaml': Cannot find module 'babel-eslint'
```
seems that eslint is trying to pick up some modules configuration

Reply:
I think running `npm install` in the library folder might be related to your other issue [here](https://github.com/open-feature/js-sdk-contrib/pull/418#discussion_r1252274079).